client/src/models/AddReview.ts

```diff
@@ -10,6 +11,9 @@ const AddReview = z.object({
     idBook: z.custom<BookViewModelType>()
         .refine((book) => book != null, "Pole wymagane")
         .transform(book => book.id),
+    idUser: z.custom<UserDetailViewModelType>()
+    .refine((user) => user != null, "Pole wymagane")
+    .transform(user => user.id),
```

Comment:
id usera nie może być tutaj, każdy może sobie wejść wtedy i to zmienić w htmlu, id użytkownika powinno być brane na backendzie z tokenu